### ISSUE TITLE
Upgrade not completing

### ISSUE THREAD

**Issue by dnmc919** [jellyfin/jellyfin#8322]
### Please describe your bug

Jellyfin 10.7.7 on Ubuntu 20.04 LTS is not upgrading through repo.  The apt update and upgrade from the repo copies the 10.8.4 packages, but the dashboard is still reporting 10.7.7.  I also attempted an apt full-upgrade, but nothing changed.

apt show jellyfin jellyfin-web jellyfin-server
![jellyfin version packages](https://user-images.githubusercontent.com/27072159/186906972-e279a938-248f-4f65-b96b-40a15e7de919.png)
Screenshots of dashboard
![jellyfin version server](https://user-images.githubusercontent.com/27072159/186907094-064faa34-d6bb-422c-bab6-d0e0a1fbf719.png)
![jellyfin version web](https://user-images.githubusercontent.com/27072159/186907113-eff5e24b-7ae2-41f7-9d24-bbb6e1657c7b.png)

The apt update shows all packages up to date, and the apt upgrade does not indicate any package or dependency issues as shown in the apt update and upgrade screenshots
![jellyfin apt update](https://user-images.githubusercontent.com/27072159/186908359-6b182af8-36db-4269-9ca1-4c0c228da0c9.png)
![jellyfin apt upgrade](https://user-images.githubusercontent.com/27072159/186908492-e8324910-5f94-47e5-8a41-a47db8e8d553.png)

The original 10.7.7 install was performed according to instructions and is running fine.  If "apt install jellyfin" is attempted, it indicates new packages will be installed, and the old removed even though the upgrade was not automatically performed from the apt update and upgrade process and the defined source.
![jellyfin apt install](https://user-images.githubusercontent.com/27072159/186908557-adb9d1b9-0e0a-4463-8436-4eb58e812d10.png)
cat /etc/apt/sources.list.d/jellyfin.list
![jellyfin source](https://user-images.githubusercontent.com/27072159/186905517-3cff8019-78f8-47c0-bb3c-efdd0b5c6658.png)

I can provide any logs that would be beneficial, but a journalctl grep of jellyfin was incredibly large, so some guidance on specific information to be queried would help minimize this to pertinent information. The jellyfin logs from the dashboard had no related errors since it is performing correctly outside of the upgrade issue.

Aside from desiring automatic updates, I am assuming the apt install could be performed, but I did not know if the existing install would be compromised by letting it remove the old package and install the new ones. Any feedback on this would be appreciated as well.

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS: Ubuntu 20.04 LTS
- Virtualization: VMWare ESXi 5.5
- Clients: NVidia Shield Android client
- Browser: Firefox on server
- FFmpeg Version: 4.4.1
- Playback Method: 
- Hardware Acceleration: none
- Plugins: Open Subtitles, Reports, TheTVDB, Trakt
- Reverse Proxy: none
- Base URL: none
- Networking: Host
- Storage: local
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Bond-009** [comment]
Did you clear your browsers cache?

**Comment by dnmc919** [comment]
Yes. And what brought the issue to light was a client upgrade of Jellyfin for android on my Nvidia Shield stopped working after the client upgrade saying it required 10.8.

**Comment by dnmc919** [comment]
This is still an issue. The "apt list jellyfin" now shows 10.8.5-1, so the updates are working in Ubuntu, but the Dashboard still shows 10.7.7 so clients with the latest update can no longer connect to the server until it reports correctly that it is on 10.8.
